node/src/accountant/big_int_db_processor.rs

```diff
@@ -1541,26 +1867,28 @@ mod tests {
         assert_eq!(high_bytes_error,
                    SqliteFailure(
                        rusqlite::ffi::Error{ code: ErrorCode::Unknown, extended_code: 1 },
-                       Some("Error from biginthigh: None negative slope, while designed only for use with negative one: 5656.23".to_string())
+                       Some("Error from slope_drop_high_bytes: Nonnegative slope '5656.23', \
```

Comment:
I may have mentioned a rewording here already, but in case I didn't:
"Nonnegative slope 5656.23; delinquency slope must be negative, since debts must become more delinquent over time."